site/en/docs/extensions/mv3/storage-and-cookies/index.md

```diff
@@ -87,7 +87,7 @@ When an extension embeds a third-party site, that site will use the extension or
 Cookies set on chrome-extension:// pages always use [`SameSite=Lax`][same-site]. Consequently, they
 can never be accessed in iframes and partitioning is not relevant.
 
-When an extension embeds a third-party website inside of a chrome://extension page, the behavior
+When an extension embeds a third-party website inside of a chrome-extension:// page, the behavior
```

Comment:
What is a chrome-extension:// page?

Reply:
There's a definition under "Update the content security policy" here: https://developer.chrome.com/docs/extensions/migrating/improve-security/